app/models/form/sales/subsections/setup.rb

```diff
@@ -7,10 +7,10 @@ def initialize(id, hsh, section)
 
   def pages
     @pages ||= [
+      Form::Sales::Pages::SaleDate.new(nil, nil, self),
```

Comment:
We can't make this depend on year, since we can't distinguish years before this is answered.